Bottom: 50f1b37136ea9fb6bc9d4df1d140fd40930209e3
Top:    c81fcb265432ca5826a4e3431824c3c9c5b5486a
Author: scott snyder <sss@karma>
Date:   2018-08-26 20:24:30 -0400

Bump target version to 20.



---

diff --git a/ComicReader/project.properties b/ComicReader/project.properties
index ea89160..92b1625 100755
--- a/ComicReader/project.properties
+++ b/ComicReader/project.properties
@@ -8,4 +8,4 @@
 # project structure.
 
 # Project target.
-target=android-8
+target=android-20
